.github/workflows/build-manylinux-container-images.yml

```diff
@@ -44,6 +44,7 @@ jobs:
         YEAR:
         - 2014
         - _2_24  # PEP 600
+        - _2_28
```

Comment:
Looks like the same tag is needed in the CI/CD workflow. This one only builds images, not the project itself.

Reply:
Ah, nevermind. #532 does that.